homeworks/maksym.selenkov_MaksymSelenkov/6-typescript/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "6-typescript",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/lodash": "^4.14.177",
+    "css-loader": "^6.5.1",
+    "style-loader": "^3.3.1",
```

Comment:
I see you have `css-loader` and `style-loader` here, but you have never used them in this HW. Are they extra? Can we delete these dependencies?